imageio/config/extensions.py

```diff
@@ -762,7 +762,7 @@ def reset(self):
     ),
     FileExtension(
         extension=".raw",
-        priority=["RAW-FI", "LYTRO-ILLUM-RAW", "LYTRO-F01-RAW"],
+        priority=["rawpy", "RAW-FI", "LYTRO-ILLUM-RAW", "LYTRO-F01-RAW"],
```

Comment:
```suggestion
        priority=["RAW-FI", "LYTRO-ILLUM-RAW", "LYTRO-F01-RAW", "rawpy"],
```

Same as my previous comment. Start in a "low-impact" setting and slowly increase.